src/main/main.js

```diff
@@ -163,7 +174,6 @@ export default class SbomGithubAction {
         let baseUrl = new URL(snInstanceUrl);
         let requestUrl = new URL(apiPath, baseUrl);
         Object.keys(uploadOptions).filter(key => uploadOptions[key]).forEach(key => requestUrl.searchParams.append(key, uploadOptions[key]));
-        requestUrl.searchParams.append('source', 'api');
```

Comment:
Removed hard coded value `api` and instead the statement on line `176` accounts for writing it if it's defined.